api-reference/docstubs/api/calendar-getschedule.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "getSchedule"
+description: ""
+author: ""
+localization_priority: Normal
+ms.prod: ""
+doc_type: apiPageType
+---
+
+# getSchedule
+
+Namespace: microsoft.graph
+
+
```

Comment:
Missing (boiler plate) description (like in the function earlier).  Maybe this is just for functions and actions topics - based on observing this in 2 files :)